src/org/zaproxy/zap/extension/httpsinfo/MDialog.java

```diff
@@ -322,4 +327,22 @@ private void raiseCrimeAlert() {
 				baseMessage); // HTTPMessage
 		extensionAlert.alertFound(alert, baseMessage.getHistoryRef());
 	}
+	
+	/**
+	 * Check if ZAP is configured to use an outbound proxy. If it is then log a 
+	 * warning and warn via a GUI dialog (if View initialised). Results may be 
+	 * inaccurate, representing the connection to the proxy instead of the 
+	 * connection to the target.
+	 */
+	private void checkProxyChainEnabled() {
+		if (Model.getSingleton().getOptionsParam().getConnectionParam().isUseProxyChain()) {
+			String warningMsg = MessageFormat.format(
+					Constant.messages.getString("httpsinfo.warn.outgoing.proxy.enabled"),
+					Constant.messages.getString("httpsinfo.name"));
+			LOGGER.warn(warningMsg);
+			if (View.isInitialised()) {
```

Comment:
Is this check really needed? This is a UI class so the view should be initialised already (if not this code should not be being called).
The log would no longer be necessary.